ssh/manifests/server/config_users.pp

```diff
@@ -0,0 +1,53 @@
+class ssh::server::config_users (
+  $users = $quickstack::params::moc_users
+) {
+
+  define config_user ($user_config = $title) {
```

Comment:
Yes.
https://docs.puppetlabs.com/puppet/latest/reference/lang_defined_types.html#title-and-name

I tried $name and $title, and I think only $title worked for me. I will try $name once the foreman is up to see if I can use it to avoid any confusion. :)